Dockerfile

```diff
@@ -1,35 +1,25 @@
-FROM node:argon-wheezy
+FROM node:latest
```

Comment:
This might cause problems on new builds; do you know which version should be used?